### PR TITLE
[FEAT] boot image information from console log data

### PR DESCRIPTION
**Pull request by sindhu-106**
Signed-off-by: sindhuja <sindhuja.gajula@intel.com>

### CODE REVIEW

File: configurations/eventsim/ProviderProvisionerNetworkForeignBus.json [hunk 1]
```diff
@@ -67,8 +67,8 @@
     "logs": "LogData",
     "events": "RasEvent",
     "stateChanges": "StateChangeEvent",
-    "HPCMLOG": "PowerLogData",
-    "SYSLOG": "DHCPLogData",
-    "BOOTIMAGE": "BootImageData"
+    "heartbeat": "PowerLogData",
+    "syslog": "DHCPLogData",
+    "consolelog": "ConsoleLogData"
```
Comment (paul-amonson) [inline configurations/eventsim/ProviderProvisionerNetworkForeignBus.json:72]: This map should be static. Why aren't you using "logs" which was designed for logs in CommonData format?  Anyway, these aren't added to the enum mapping in DataType in the DAI Network Listener component.  As a side note the plan is to move this to resources (just haven't got to it yet).

**Review by paul-amonson — CHANGES_REQUESTED** [review]

**Review by paul-amonson — DISMISSED** [review]

File: configurations/eventsim/ProviderProvisionerNetworkForeignBus.json (hunk 1 above)
Comment (paul-amonson) [reply configurations/eventsim/ProviderProvisionerNetworkForeignBus.json]: Handle this with a separate review. :)

**Review by yhuan47 — DISMISSED** [review]

File: configurations/eventsim/ProviderProvisionerNetworkForeignBus.json (hunk 1 above)
Comment (sindhu-106) [reply configurations/eventsim/ProviderProvisionerNetworkForeignBus.json]: sure Paul

**Review by paul-amonson — APPROVED** [review]

**Review by avincigu — APPROVED** [review]

**Comment by paul-amonson** [comment]
I will be merging this without the checks completed. It seems travis-ci.org was deprecated. Working on getting travis-ci.com functioning.

**Pull request merged**